nucypher/characters/lawful.py

```diff
@@ -670,12 +672,12 @@ def work_orders_for_capsules(self,
                 raise KeyError(f"Bob doesn't have the TreasureMap {map_id}; can't generate work orders.")
 
         incomplete_work_orders = OrderedDict()
-        complete_work_orders = OrderedDict()
+        complete_work_orders = defaultdict(lambda: [])
```

Comment:
why the `lambda` here?

Reply:
```suggestion
        complete_work_orders = defaultdict(list)
```

Reply:
Yeah, must have been brain freeze. Fixed.